examples/specs/test_subobject_missing.vl.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "Unable to truly filter data",
```

Comment:
This description is very confusing.   

The description suggests that this spec does NOT work, but it seems to be working.  So I don't understand what's the point you're trying to make here.   (What is "truly filter" data?)

Reply:
Ups, copy pasta.